src/scenes/levelend-scene/levelend-scene.ts

```diff
@@ -142,6 +118,70 @@ export class LevelEndScene {
     }
   }
 
+  private createButton(
+    ButtonClass: typeof MapButton | typeof RetryButtonHtml | typeof NextButtonHtml,
+    id: string,
+    onClickCallback: () => void
+  ) {
+    const buttonsContainerId = 'buttons-container';
+
+    const button = new ButtonClass({ targetId: buttonsContainerId, id });
+    const gameControl = document.getElementById("game-control") as HTMLCanvasElement;
+    button.onClick(() => {
+      this.levelEndElement.style.zIndex = '7';
+      gameControl.style.zIndex = "-1";
```

Comment:
Can we add comments on what does this zIndex do? They very common in our code base and it can be quite confusing reading them as they don't have much context.

Reply:
added comments @bernhardccodev ? thank you sir